bin/run_tests.sh

```diff
@@ -4,6 +4,13 @@ set -e
 base=$(pwd)
 failed=0
 
+# run snyk tests locally. Skip in bamboo due to authentication token requirement.
```

Comment:
What stops this from being run in bamboo? I don't recognize the branch point.

Reply:
if we also want to run these snyk tests in bamboo, then we have to create a variable for passing the snyk authentication token and the pipeline will have to be run manually which is extra step.
I also noticed there is already a stage thats runs snyk scan in bamboo.
See https://github.com/nasa/cumulus-orca/blob/develop/bamboo-specs/bamboo.yaml#L96

Reply:
But we run this script in Bamboo. So when this script is run in Bamboo, what enforces the "Skip in bamboo due to authentication token requirement"? I don't see an if-check or other branch point.

Reply:
I think will still look if there is a built-in bamboo variable that I can use and then use if else condition.

Reply:
Per our conversation, recommend moving tests for website into a separate script we can run manually locally. For tests on IAC we need to follow up with Jon V and check on licensing. I know last month, they were limited to only 20 or so runs a month on IAC checks. We will need to see if this is still true or if it is now unlimited. I also recommend that we keep the check as a separate script that we can run manually. 

Since we already check things in Bamboo via snyk, recommend that we would only run theses locally.

Reply:
after discussion with team, we now have a separate script for snyk that will have to be manually run in our local machine.

Reply:
after discussion with team, we now have a separate script for snyk that will have to be manually run in our local machine.

Reply:
plus we have decided not to run it in bamboo since bamboo already has snyk job added